.env.example

```diff
@@ -0,0 +1,38 @@
+# Default values for Azure OpenAI 
+AZURE_OPENAI_EMBEDDING_DEPLOYMENT_NAME=text-embedding-ada-002
+AZURE_OPENAI_EMBEDDING_MODEL_NAME=text-embedding-ada-002
+AZURE_OPENAI_PROMPT_DEPLOYMENT_NAME=gpt-35-turbo
+AZURE_OPENAI_PROMPT_MODEL_NAME=gpt-35-turbo
+AZURE_OPENAI_DEFAULT_ENDPOINT=https://griptape-cloud-nonprod-scus.openai.azure.com/
+
+# Get from Azure Portal https://azure.microsoft.com/en-us/get-started/azure-portal
```

Comment:
Good to tell me where to get them, but what do I need them for? What do they empower in the app?